git_machete/exceptions.py

```diff
@@ -12,3 +12,7 @@ def __str__(self) -> str:
 class StopTraversal(Exception):
     def __init__(self) -> None:
         pass
+
+
+class UnprocessableEntityError(Exception):
```

Comment:
Is this error needed? There is in reality no error like that, there is just generic HTTPError that have some error code inside. By extracting this status code as separate exception we're loosing the context of HTTP Error. As it is now it's hard to figure out what this error is about (what is entity? database entry or maybe element of an array?).

I propose following options:
1. Add docstring specifying that this is HTTP Error with status code 422. Ideally also add context to this exception by either putting into the file `http_errors.py` or by storing this exception in some dict (`http_specific_errors`?) and than referring to it by `raise http_specific_errors[http.HTTPStatus.UNPROCESSABLE_ENTITY]`
2. Don't implement this exception and just use HTTP Error instead with appropriate msg etc.

Reply:
(resolved?)

Reply:
Since context has been added to this exception, i think so, yes. @BartoszSiwinski, Could you confirm?